src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -165,17 +182,16 @@ class SentryAppExternalIssueForm extends React.Component {
             <FieldFromConfig
               key={`${field.name}`}
               field={field}
-              inline={false}
-              stacked
-              flexibleControlStateSize
-              required
               {...this.fieldProps(field)}
             />
           );
         })}
 
         {optionalFields.map(field => {
           field.choices = field.choices || [];
+          field.inline = false;
+          field.stacked = true;
+          field.flexibleControlStateSize = true;
```

Comment:
Nice that TS found some problems :clap:

Reply:
@markstory Actually, not really in this case. We end up applying the extra props to the field in the child component anyways. This change is just more TS friendly but functionally the same.